bundles/InstallBundle/Resources/install.sql

```diff
@@ -501,6 +501,9 @@ CREATE TABLE `tags` (
   `parentId` int(10) unsigned DEFAULT NULL,
   `idPath` varchar(190) DEFAULT NULL,
   `name` varchar(255) DEFAULT NULL,
+  `translations` text DEFAULT NULL,
```

Comment:
migration is missing

Reply:
Omg, you're right.. I added them immediatly...